cmd/upcloud-csi-manifest/main.go

```diff
@@ -58,6 +59,7 @@ func main() { //nolint: funlen // TODO: refactor
 	if *setupManifest {
 		templates = append(templates, kubernetes.CSITemplate)
 		vars["UPCLOUD_CSI_VERSION"] = *driverVersion
+		vars["CLUSTER_ID"] = *labelClusterID
```

Comment:
No trouble if we don't check for nil values? Are they converted to empty strings before being assigned into labels?

Reply:
@ajmyyra  thanks for review :+1: 
I think that pointer should always have some value, as we provide default (empty string) to `flagSet.String`.